docs/maya/skin_merge.md

```diff
@@ -0,0 +1,66 @@
+# AdnSkinMerge
+
+AdnSkinDeformer is a Maya deformer to merge animation geometries with the results provided by AdnSkin. This deformer allows for the merging of several animation and simulation meshes into a single final mesh.
+
+The influence simulation or animation meshes will have on the final mesh can be freely painted and modified by painting a blends weight map.
```

Comment:
Would this be more correct (I may  be wrong)?
```suggestion
The influence simulation or animation meshes will have on the final mesh can be freely painted and modified by painting a blend weight map.
```

Reply:
Corrected!